test/data/stages/xz/b.mpp.json

```diff
@@ -28,8 +21,9 @@
             "file": {
               "type": "org.osbuild.files",
               "origin": "org.osbuild.source",
-              "references": {
-                "sha256:f950375066d74787f31cbd8f9f91c71819357cad243fb9d4a0d9ef4fa76709e0": {}
+              "mpp-embed": {
+                "id": "hw",
+                "text": "Hallo Welt\n"
```

Comment:
Don't we also want to demo/test referencing the `embedded['hw']` variable?

Reply:
Good point, I think the tar stage test uses the copy stage.